moped-database/migrations/1621965215544_create_table_public_moped_subcomponents/up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE "public"."moped_subcomponents" (
+    "subcomponent_id" serial NOT NULL,
+    "subcomponent_name" text NOT NULL,
+    "component_id" integer NOT NULL,
+    PRIMARY KEY ("subcomponent_id") ,
+        FOREIGN KEY ("component_id")
+            REFERENCES "public"."moped_components" ("component_id")
+                ON UPDATE cascade ON DELETE cascade
+);
+
+INSERT INTO moped_subcomponents (
```

Comment:
@sergiogcx we're going to want to change these values quite soon—but i don't see any reason to worry about the values until we deploy this new schema.